docs/sphinx/sections/api/apidocs/schedules-sensors.rst

```diff
@@ -16,32 +27,19 @@ Schedules
 
 .. autofunction:: build_schedule_context
 
+.. autofunction:: build_schedule_from_partitioned_job
+
 .. currentmodule:: dagster._core.scheduler
 
 .. autoconfigurable:: DagsterDaemonScheduler
   :annotation: Scheduler
 
-Partitioned Schedules
-=====================
-
-.. currentmodule:: dagster
-
-.. autofunction:: build_schedule_from_partitioned_job
-
-.. autodecorator:: hourly_partitioned_config
-    :noindex:
-
-.. autodecorator:: daily_partitioned_config
-    :noindex:
-
-.. autodecorator:: weekly_partitioned_config
-    :noindex:
-
-.. autodecorator:: monthly_partitioned_config
-    :noindex:
+----
 
 Sensors
-=======
+-------
+
+`Sensors </concepts/partitions-schedules-sensors/sensors>`_ allow you to run a job or materialize an asset in response to specific events. Sensors periodically check and execute logic to know whether to kick off a run.
```

Comment:
Technically, schedules also periodically check and execute logic to know whether to kick off a run.

thinking about what separates a schedule and a sensor, there are at least two key things:
* the high frequency aspect of it
* you mentioned it, but sensors are conventionally used to poll, listen, and respond to external events. because the high frequency and cursor aspects are better suited to do this over schedules.

Reply:
I updated this a bit, but I don't want to go too far into explaining sensors in the API reference. We can come back to this when the Sensor overhaul takes place.